dev/integration_tests/flutter_gallery/pubspec.yaml

```diff
@@ -169,7 +169,6 @@ flutter:
     - packages/flutter_gallery_assets/products/table.png
     - packages/flutter_gallery_assets/products/teaset.png
     - packages/flutter_gallery_assets/products/top.png
-    - packages/flutter_gallery_assets/people/ali.png
```

Comment:
Reminder to add this back

Reply:
I deleted this line because `people/ali.png` doesn't exist. I suspect this wasn't an issue before because there is a `people/square/ali.png`, and the variant detection code, when searching for `people/ali.png`, would see `people/square/ali.png` as a 1x variant of `people/ali.png`.